### ISSUE TITLE
[New Page is showing after uploading File Successfully]

### ISSUE THREAD

**Issue by tailm1** [orma/openroads-vn-analytics#581]
Step 1 : Go to Upload Page 
Step 2 : Upload correct file 
Expected Result : a successful dialog is showing to notice new information
Actual : New page/tab is display

**Comment by geohacker** [comment]
Thank you for opening the issue. 

This is the legacy behavior of RLP properties upload. Particularly because the file upload is handled as POST request directly instead of an ajax request via the front-end. The RLP geometries uploads are rebuilt to allow for conflation and hence follow a job based architecture. 

We'll attempt to add a success/error page but the new tab will show the current state of the upload. 

cc @batpad

**Comment by batpad** [comment]
@geohacker yes, I think we would need to treat changing the architecture of the properties upload as an enhancement. However, we should definitely improve the error handling when there is invalid / no data in the uploaded file, like in this case: https://github.com/orma/openroads-vn-analytics/issues/567

Can make a ticket to improve error message after properties upload for invalid data.
